### PR TITLE
JSUI-1799 Changing remap-istanbul to a forked version and simplifying regex

### PR DESCRIPTION
**Pull request by samisayegh**
https://coveord.atlassian.net/browse/JSUI-1799

[Deploy](https://dashboard.heroku.com/pipelines/a3535101-5bbf-4a5b-a909-47fcf8c9f149)

### CODE REVIEW

**Comment by samisayegh** [comment]
Here is the forked  version of [remap-istanbul](https://github.com/coveord/remap-istanbul). I submitted a PR that has already been accepted [here](https://github.com/SitePen/remap-istanbul/pull/152), so we should be able to use the official library on the next release, and delete the fork.

Also, please check if the command is compatible with Linux and Mac by running `gulp coverage`.

File: gulpTasks/test.js [hunk 1]
```diff
@@ -66,3 +67,8 @@ gulp.task('lcovCoverage', ['remapCoverage'], function (done) {
     print: 'summary'
   }).then(() => done());
 });
+
+function filesToExclude(fileName) {
+  const whiteList = /search-ui[\/\\](bin|src|test)/;
```
Comment (dms1lva) [inline gulpTasks/test.js]: We need files from the `bin` folder? :thinking:

File: package.json [hunk 2]
```diff
@@ -20,19 +20,43 @@
     "uploadCoverage": "gulp uploadCoverage"
   },
   "lint-staged": {
-    "*.{js,ts,json}": ["prettier --single-quote --print-width 140 --write", "git add"]
```
Comment (dms1lva) [inline package.json]: Is it possible that you ran prettier without the printWidth: 140 parameter? Maybe with an extension in your editor? This seems to be less than 140 characters. It is possible to add this config in the package.json like this
```  "prettier": {
    "printWidth": 140,
    "singleQuote": true
  }```

I will add it in my next PR but if you need it now, might as well.

Comment (olamothe) [reply package.json]: I think it might be vscode auto formatting ? 

Not sure why it's not getting reformatted properly with the precommit hook though 🤔

**Pull request merged**